### PR TITLE
Added meta.yaml for snakeviz

### PR DESCRIPTION
**Pull request by Bergvca**
Add snakeviz to the recipes

### CODE REVIEW

**Comment by Bergvca** [comment]
@CJ-Wright

File: recipes/snakeviz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```
Comment (CJ-Wright) [inline recipes/snakeviz/meta.yaml]: this needs requests and tornado as dependencies

**Comment by Bergvca** [comment]
@CJ-Wright added dependencies

**Comment by saraedum** [comment]
@CJ-Wright This is a bit old already. Just curious why this was never merged?

**Comment by CJ-Wright** [comment]
I think this looks ok, I don't know why I didn't merge it.

**Pull request merged**